tests/impls/vis_sal_classifier/test_occlusion_classifier.py

```diff
@@ -0,0 +1,83 @@
+from unittest import TestCase
+
+import numpy as np
+
+from xaitk_saliency.impls.vis_sal_classifier.occlusion_scoring import OcclusionScoring
+from xaitk_saliency import ImageClassifierSaliencyMapGenerator
+
+
+class TestOcclusionBasedScoring (TestCase):
+
+    def test_init_(self) -> None:
+        """
+        Test if implementation is usable.
+        """
+        impl = OcclusionScoring()
+        assert impl.is_usable() and isinstance(impl, ImageClassifierSaliencyMapGenerator)
+
+    def test_1class_scores(self) -> None:
+        """
+        Test basic scoring with a single class for broadcasting sanity check.
+        """
+        impl = OcclusionScoring()
+        # Three Pertrubation masks of height and width 10px for 1 class
+        image_confs_1_class_ = np.random.rand(1)
```

Comment:
When using random values in unit tests, I'd recommend using seeded generator for reproducibility.

Reply:
Great point. Added a seed

Reply:
Looks good. I don't request the following, but FYI there is another method of stabilly performing seeded random operations isloated from other uses of random: np.random.RandomState(seed: int) which results in an object that you can do all the normal random function calls on. This can isolate the random seeding to that new state and not affect other, e.g. nested, uses of random.